.github/workflows/update-config.yml

```diff
@@ -0,0 +1,84 @@
+name: Pull Request Action
+permissions: write-all
+on:
+  push:
```

Comment:
Pls remove the `push` handler, we don't want to spam with PRs

Reply:
should we do this on workflow_dispatch?

Reply:
or 
```
on:
  push:
    paths:
    - 'ubiquibot-config.-default.json'
```